src/util.h

```diff
@@ -29,6 +29,9 @@
 #include <boost/signals2/signal.hpp>
 #include <boost/thread/exceptions.hpp>
 
+// Application startup time (used for uptime calculation)
+static const int64_t nStartupTime = GetTime();
```

Comment:
The `DEFAULT_LOG` constants are [already in master](https://github.com/bitcoin/bitcoin/blob/master/src/util.h#L33). Might be something to improve then.

Also, I don't know if you noticed, this variable was [moved](https://github.com/bitcoin/bitcoin/blob/master/src/qt/clientmodel.cpp#L29) from `src/qt/clientmodel.cpp` to a more global area so that it can be reused by both the server and the QT client. 

I will make this change as you suggested and force-push the commit.